packages/main/test/specs/ComboBox.mobile.spec.js

```diff
@@ -151,15 +155,39 @@ describe("Eventing", () => {
 		await dialogOkButton.click();
 
 		assert.strictEqual(await combo.getAttribute("value"), "Argentina", "The combo box have correct value attribute");
-
 		const changeText = await browser.$("#change-placeholder").getText();
-		assert.strictEqual(changeText, "Argentina", "The field was changed as change event was fired");
 
+		assert.strictEqual(changeText, "Argentina", "The field was changed as change event was fired");
 		const changeCountText = await browser.$("#change-count").getText();
-		assert.strictEqual(changeCountText, "1", "Change was fired once");
 
+		assert.strictEqual(changeCountText, "1", "Change was fired once");
 		assert.strictEqual(await combo.getValue(), "Argentina", "The original value was changed");
-	}); */
+		
+	});	*/
+
+	it ("When select an item, then open the dialog again and delete the text, then press OK button, the value should be deleted.", async ()=> {
+		const cb = await browser.$("#combo2");
+		const staticAreaItemClassName = await browser.getStaticAreaItemClassName("#combo2");
+		
+		await cb.click();
+	
+		const resPopover = await browser.$(`.${staticAreaItemClassName}`).shadow$("ui5-responsive-popover");
+		const dialogInput = await resPopover.$(".ui5-input-inner-phone");
+		const okBtn = await resPopover.$(".ui5-responsive-popover-footer").$("ui5-button");
+		await dialogInput.click();
+		await dialogInput.setValue('');
+		await dialogInput.keys('A');
+		await dialogInput.keys('ArrowDown');
+		await okBtn.click();
+
+		assert.strictEqual(await cb.getProperty("value"), "Algeria", "Value should be Algeria.");
+		await cb.click();
+		await dialogInput.click();
```

Comment:
You can deleted line 185 and 186 and the result will be the same, the input in the dialog opens with selected value.